.rubocop_todo.yml

```diff
@@ -9,9 +9,15 @@
 # Offense count: 1
 # Configuration parameters: IgnoredMethods, CountRepeatedAttributes.
 Metrics/AbcSize:
+  Max: 20
+
+# Offense count: 1
+# Configuration parameters: CountComments, CountAsOne, ExcludedMethods, IgnoredMethods.
+# IgnoredMethods: refine
+Metrics/BlockLength:
```

Comment:
it's not super important, but adding a new offense with a count of 1 might give me pause to see if something can be refactored AND achieve better clarity.   No biggie.

If it's the spec, you can configure rubocop to ignore.